server/controllers/report.js

```diff
@@ -474,7 +475,7 @@ function cotisation_payment(params) {
     'JOIN paiement p ON e.id=p.employee_id ' +
     'JOIN cotisation_paiement z ON z.paiement_uuid=p.uuid ' +
     'JOIN cotisation t ON t.id=z.cotisation_id ' +
-    'WHERE p.paiement_period_id= ' + id +
+    'WHERE p.paiement_period_id= ' + id + ' AND z.cotisation_id=' + cotisation_id +
```

Comment:
It would be better to use db.exec() for paramter escaping:

```js
'WHERE p.paiement_period_id = ? AND z.cotisation_id = ?;';

db.exec(sql, [id, cotisation_id])
```